src/FAQPage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import FAQPage_data from './data/FAQPage_data.js';
+
+const DisplayAnswere = (Props) => {
+  return Props.DisplayAns ? <p id="a1"> {Props.Answ} </p> : <p></p>;
+};
+
+const FAQItem = (props) => {
+  const { Ques, Answ } = props.itemData;
+  const [displayAns, toggleDisplayAns] = useState(false);
+
+  return (
+    <dl>
+      <dt>
+        <button
+          aria-expanded="true"
+          aria-controls="a1"
+          onClick={() => toggleDisplayAns(!displayAns)}
+        >
+          {Ques}{' '}
+        </button>
+      </dt>
+
+      <dd>
+        <DisplayAnswere DisplayAns={displayAns} Answ={Answ} />
+      </dd>
+    </dl>
+  );
+};
+
+export const FAQPage = () => {
+  return (
+    <div>
+      <h2>FAQ Page </h2>
+      FAQPage_data.map((data) => {<FAQItem key={id} itemData={data} />})
```

Comment:
Hello,
in order to use FAQPage_data.js inside jsx you should use it inside brackets.
for example {FAQPage_data.map(//...)}
and the other issue is that map function expects a return, the arrow function with '{}' is not returning the <FAQItem/> component

you can use it like
{ FAQPage_data.map((data) => { return (<FAQItem key={id} itemData={data} />) } ) }

or

{FAQPage_data.map((data) => (<FAQItem key={id} itemData={data} />))}  you don't even need '()' for one line returns

I hope this helps. :)

Reply:
thanks @pnzrkmpfwgn it helped.